cristin-biobank/src/main/java/no/unit/nva/biobank/model/cristin/CristinCoordinatinInstitution.java

```diff
@@ -0,0 +1,30 @@
+package no.unit.nva.biobank.model.cristin;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import no.unit.nva.cristin.model.CristinInstitution;
+import no.unit.nva.cristin.model.CristinUnit;
+
+public class CristinCoordinatinInstitution {
```

Comment:
What about this:
https://github.com/BIBSYSDEV/nva-cristin-service/blob/main/cristin-commons/src/main/java/no/unit/nva/cristin/model/CristinOrganization.java
?

Reply:
Originally I was planning somehow to use it from commons,
But then Ketil said that in NVA model biobank we just need URI of the organization and unit + it is not all information about organization which is filled. 
Maybe it is more reasonable to use organization still, true. 
Need second opinion as you said.

Reply:
You can use CristinOrganization and populate only id (uri) too.